lib/tessel/setup-3g.js

```diff
@@ -0,0 +1,119 @@
+var Tessel = require('./tessel');
```

Comment:
Can you update the use of `var` in this file to `const`? Those variables are never reassigned so we can optimize with the use of `const`.